doc/source/whatsnew/v1.3.2.rst

```diff
@@ -15,25 +15,25 @@ including other versions of pandas.
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
 - Performance regression in :meth:`DataFrame.isin` and :meth:`Series.isin` for nullable data types (:issue:`42714`)
-- Regression in updating values of :class:`pandas.Series` using boolean index, created by using :meth:`pandas.DataFrame.pop` (:issue:`42530`)
```

Comment:
should there be a pygrep check for
```
:class:`pandas`
```
if this is a common enough occurrence?

Reply:
I think people look at the adjacent code for inspiration, so maybe it would be common if let through when the release notes don't have many entries.

although no harm in a check.

Reply:
we probably want to allow `pandas.NA`